src/Calculator.tsx

```diff
@@ -32,30 +32,30 @@ class Calculator extends React.Component {
      * 
      * @param {} recallWeight 
      */
-    getFScore = (recallWeight) => {
+    getFScore = (recallWeight: number) => {
```

Comment:
> src/Calculator.tsx:32:18 - error TS7006: Parameter 'recallWeight' implicitly has an 'any' type.

TypeScript recommends that you add type annotations to function parameters, to help catch bugs. This is optional, and you can disable this by turning off `noImplicitAny` in tsconfig (which I can teach you how to).